### DISCUSSION TITLE
F1: Keyboard shortcut to adjust volume of selected feed, with visual display of selected volume

### DISCUSSION THREAD

**Discussion by blademan007** [f1multiviewer/issue-tracker#426] [Feature requests]
F1MV with smaller sized feeds, it's hard to eyeball the volume slider. Would recommend ability to adjust in 5 point increments.
Maybe additional feature request, but adding it here: visual display of current volume when making adjustments

**Comment by jeppeolesen** [comment]
Redirecting this to #344
